src/helm/proxy/clients/mechanical_turk_critique_exporter.py

```diff
@@ -0,0 +1,194 @@
+import csv
+import os
+from threading import Lock
+from typing import Dict, List, Sequence
+import textwrap
+import re
+
+from helm.common.critique_request import CritiqueQuestionTemplate, CritiqueRequest, CritiqueTaskTemplate, QuestionType
+from helm.common.general import ensure_directory_exists
+from helm.common.hierarchical_logger import hlog
+
+
+def _indent_to_level(text: str, level: int) -> str:
+    """Helper for indenting XML to the same level as the external template."""
+    return textwrap.indent(text, " " * 4 * level).lstrip()
+
+
+def _format_template_tags(raw_text: str) -> str:
+    """Convert from Surge AI template tag format to Mechanical Turk template tag format.
+
+    {{field}} -> ${field}"""
+    return re.sub(r"{{([^{}]+)}}", "${\\1}", raw_text)
```

Comment:
I tested it out `summarization_cnndm:temperature=0.3,device=cpu,model=openai_ada` with just 5 instances and currently the output CSV file looks like this:

<img width="950" alt="Screenshot 2023-05-09 at 1 11 39 PM" src="https://github.com/stanford-crfm/helm/assets/16793796/5e0ea36f-368b-4102-91c8-f35c7b1cd157">

I think maybe new lines are not handled well?

Reply:
The formatting is correct; here's what it looks like after importing into Mechanical Turk. (I added more newlines for illustration.)

Note that newlines in raw HTML are not displayed the end user (this is just how HTML works). You will have to use `<p></p>`, `<br>` or `style="white-space: pre-wrap;"` if you want to render newlines.

![Screenshot 2023-05-09 154717](https://github.com/stanford-crfm/helm/assets/185227/c04446cf-485a-41d1-8e42-de39b1e7009b)

Reply:
I see, that makes sense.. thanks!